secretstores/hashicorp/vault/vault.go

```diff
@@ -166,7 +166,7 @@ func (v *vaultSecretStore) Init(metadata secretstores.Metadata) error {
 		}
 	}
 
-	if !convertedVaultKVUsePrefix {
+	if convertedVaultKVUsePrefix {
```

Comment:
@shivanisinghnitp this change looks similar to the existing one. Is default value of `convertedVaultKVUsePrefix` being read somewhere else. Please make sure this is tested out.

Reply:
Should this line remain unchanged?